standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestObjectStore.java

```diff
@@ -393,6 +393,14 @@ public void testPartitionOps() throws Exception {
     try (AutoCloseable c = deadline()) {
       objectStore.addPartition(part2);
     }
+    List<String> value3 = Arrays.asList("US", "MA");
```

Comment:
Can you add test cases for the other changed API methods too? Maybe create a new method for them, e.g. `testPartitionOpsWhenTableDoesNotExist`? This test method already seems quite messy and cluttered so maybe it's worth separating it out